service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/docmosis/TranslationRequestFormCreationServiceDocmosisTest.java

```diff
@@ -35,10 +34,13 @@
     private static final String CONTACT_INFORMATION = "contactfpl@justice.gov.uk";
     private static final String FAMILY_MAN_CASE_NUMBER = "FamilyManCaseNumber123";
     private static final LocalDate FIXED_DATE = LocalDate.of(2021, 7, 28);
-    private static final int WORD_COUNT = 2034;
+    private static final long WORD_COUNT = 2034;
```

Comment:
This seems like could probably stay as int

Reply:
the stream.count() in the wordCalculator returns normally a long, so rather than downcasting to an int (casting is always ugly) I've change it to long.